test/07_test_panel.jl

```diff
@@ -34,60 +34,55 @@ end
 
 @testset "\e[34mPANEL - fit - measure" begin
     for style in ("default", "red", "on_blue")
-        for justify in (:left, :center, :right)
-            # ----------------------------- text only content ---------------------------- #
-            testpanel(Panel("t"; fit = true, style = style), 7, 3)
-            testpanel(Panel("test"; fit = true, style = style), 10, 3)
-            testpanel(Panel("1234\n123456789012"; fit = true, style = style), 18, 4)
-            testpanel(Panel("나랏말싸미 듕귁에 달아"; fit = true, style = style), 28, 3)
-            testpanel(
-                Panel("나랏말싸미 듕귁에 달아\n1234567890123456789012"; fit = true, style = style),
-                28,
-                4,
-            )
-            testpanel(
-                Panel("."^500; fit = true, style = style),
-                displaysize(stdout)[2] - 1,
-                nothing,
-            )
+        # ----------------------------- text only content ---------------------------- #
```

Comment:
No, this is because you don't `justify` in these, so this is redundant to put them in the inner loop. As a side note, I find the tests hard to read because of the duplication of the test code. I think it is worth some simplifications.

Reply:
I've added `Base.size(m::Measure)` returning `(m.w, m.h)` for that purpose.